apps/playground/src/app/modules/components/tile-module/tile-module.component.html

```diff
@@ -65,168 +64,220 @@ <h2>DsfrTileComponent</h2>
       <mat-icon>open_in_new</mat-icon>
```

Comment:
And this on line 38 : 
`The title of the tile, it's displayed in an H3 tag.`

Also add this after the `play-input-property-definition` on `align` : 
```html
<play-input-property-definition
      name="size"
      default="md"
      type="It takes a string with 3 possible values : <code>sm</code>, <code>md</code> or <code>lg</code>">
      You can import the <code>ElementSize</code> enum, from the
      <code>&#64;betagouv/ngx-dsfr</code> library entry-point, in your app, to
      safely provide a value for this property.
    </play-input-property-definition>
```